pkg/aws/core.go

```diff
@@ -171,6 +176,38 @@ func (d *Driver) CreateMachine(ctx context.Context, req *driver.CreateMachineReq
 	return response, nil
 }
 
+//returns Placement Object required in ec2.RunInstancesInput
+func getPlacementObj(req *driver.CreateMachineRequest) *ec2.Placement {
```

Comment:
From what i see, I understand that the method returns an placement object with tenancy and affinity set as `Default` in the normal case. Just wondering what this maps to in the AWS context? Could it be the case that AWS doesn't understand the placement object if there are no dedicated hosts in the account? We will need to cross check this with the documentation. 

An alternative approach could be to return a `nil` object when nothing is set. And to handle this nil object to not add this entry in the final creation request. This would let the current code as it is today and only add the placement object if anyone of the fields are set.